### PR TITLE
Updated docs on Secondary Index with JSONB datatype

### PR DESCRIPTION
**Pull request by KavyaShivashankar**
1.   Create new pages for secondary indexes with jsonb for ysql and ycql 
2.  Updated the correct link in explore json documents
3.  Updated link in ycql features to point to secondary index with JSONB pages

### CODE REVIEW

File: docs/content/preview/explore/indexes-constraints/partial-index-ycql.md [hunk 1]
```diff
@@ -135,4 +135,6 @@ ycqlsh:example> EXPLAIN SELECT product FROM orders
 
 ## Learn more
 
-For more details, refer to [PARTIAL INDEX](../../../api/ycql/ddl_create_index/#partial-index) in the YCQL API documentation.
+- For more details, refer to [PARTIAL INDEX](../../../api/ycql/ddl_create_index/#partial-index) in the YCQL API documentation.
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/partial-index-ycql.md:138]:
```suggestion
- [PARTIAL INDEX](../../../api/ycql/ddl_create_index/#partial-index) in the YCQL API documentation
```

File: docs/content/preview/explore/indexes-constraints/partial-index-ycql.md [hunk 2]
```diff
@@ -135,4 +135,6 @@ ycqlsh:example> EXPLAIN SELECT product FROM orders
...
+
+- [Partial index with JSONB column](../secondary-indexes-with-jsonb-ycql/#partial-index).
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/partial-index-ycql.md:140]:
```suggestion
- [Partial index with JSONB column](../secondary-indexes-with-jsonb-ycql/#partial-index)
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+The following section describes secondary indexes with JSONB column in YCQL using examples.
+
+{{% explore-setup-single %}}
+
+## Create table
+
+Secondary indexes in YCQL can be created only on tables with `transaction = {'enabled':'true'}`. This is due to the use of distributed ACID transactions under the hood to maintain the consistency of secondary indexes in YCQL.
+
+Any attempt to create secondary index on a table without `transactions = {'enabled':'true'}` will result in an error.
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:41]:
```suggestion
Any attempt to create a secondary index on a table without `transactions = {'enabled':'true'}` results in an error.
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+For additional information regarding the DESCRIBE INDEX command, see [DESCRIBE INDEX](../../../admin/ycqlsh/#describe).
+
+You can also use the `EXPLAIN` statement to check if a query uses an index and determine the query plan before execution.
+
+```cql
+EXPLAIN SELECT * FROM users WHERE address->>'zip' = '327';
+```
+
+For additional information, see [EXPLAIN](../../../api/ycql/explain/) statement.
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:102]:
```suggestion
For additional information, see the [EXPLAIN](../../../api/ycql/explain/) statement.
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+## Covering index and Partial index with JSONB column
+
+You can also create a covering index and partial index with a JSONB column.
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:106]:
```suggestion
You can also create covering and partial indexes with a JSONB column.
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+### Covering index
+
+Covering index includes all columns used in the query in the index definition. This is done by using the `INCLUDE` keyword in the `CREATE INDEX` syntax as follows:
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:110]:
```suggestion
A covering index includes all columns used in the query in the index definition. You do this using the `INCLUDE` keyword in the `CREATE INDEX` syntax as follows:
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+```cql
+CREATE INDEX idx_users_jsonb_cov ON users((address->>'zip'))
+     INCLUDE (first_name,last_name);
+```
+
+### Partial index
+
+Partial index is created on a subset of data when you want to restrict index to a specific condition. This is done by using `WHERE` clause in the `CREATE INDEX` syntax as follows:
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:119]:
```suggestion
A partial index is created on a subset of data when you want to restrict the index to a specific condition. You do this using the `WHERE` clause in the `CREATE INDEX` syntax as follows:
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+``` cql
+CREATE INDEX idx_users_jsonb_part ON users (address->>'zip')
+       WHERE email = 'lskywalker@yb.com';
+```
+
+For additional information on CREATE INDEX statement, see [CREATE INDEX](../../../api/ycql/ddl_create_index/).
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:126]:
```suggestion
For additional information on the CREATE INDEX statement, see [CREATE INDEX](../../../api/ycql/ddl_create_index/).
```

File: docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+## Remove indexes
+
+You can remove index created with JSONB datatype column using the `DROP INDEX` statement in YCQL with the following syntax:
```
Comment (ddhodge) [inline docs/content/preview/explore/indexes-constraints/secondary-indexes-with-jsonb-ycql.md:130]:
```suggestion
You can remove an index created with the JSONB datatype column using the `DROP INDEX` statement in YCQL with the following syntax:
```

**Review by ddhodge — APPROVED** [review]

**Review by aishwarya24 — APPROVED** [review]
pushed changes and applied the ones @ddhodge suggested. @KavyaShivashankar, its good to merge. Thanks!

**Pull request merged**